org.eclipse.xtext.xbase.testing/src/org/eclipse/xtext/xbase/testing/CompilationTestHelper.java

```diff
@@ -212,6 +226,16 @@ public void compile(Iterable<? extends CharSequence> sources, IAcceptor<Result>
 	public void compile(final ResourceSet resourceSet, IAcceptor<Result> acceptor) {
 		try {
 			List<Resource> resourcesToCheck = newArrayList(resourceSet.getResources());
+			if (generatorConfigProvider instanceof GeneratorConfigProvider) {
+				GeneratorConfigProvider configProvider = (GeneratorConfigProvider) generatorConfigProvider;
+				GeneratorConfig config = generatorConfigProvider.get(null);
```

Comment:
@svenefftinge but if someone provides a custom implementation of GeneratorConfigProvider that would not be used (I also added a test for that https://github.com/eclipse/xtext-extras/pull/47/files#diff-c477e770a10cf16ed4e0ec0c77586b3bR161). I would have wanted to pass the resourceSet to the get method, but get() takes an EObject... the alternative would have been to pass the first EObject of the first resourc ein the resourceSet.